### ISSUE TITLE
Unable to load kvsplugin

### ISSUE THREAD

**Issue by sauravkumar-volt** [aws-samples/amazon-kinesis-video-streams-demos#241]
Platform : x86_64
OS: Ubuntu 20.04

Issue: After building the plugin, and exporting the GST_PLUGIN_PATH, inspecting the ```kvsplugin``` gives these :
```
gst-inspect-1.0 kvsplugin

(gst-plugin-scanner:448006): GStreamer-WARNING **: 14:02:31.313: Failed to load plugin '/home/<username>/amazon-kinesis-video-streams-demos/gst/gst-kvs-plugin/build/_deps/cproducer-src/open-source/lib/libssl.so': /home/<username>/amazon-kinesis-video-streams-demos/gst/gst-kvs-plugin/build/_deps/cproducer-src/open-source/lib/libssl.so: undefined symbol: EVP_idea_cbc, version OPENSSL_1_1_0

(gst-plugin-scanner:448006): GStreamer-WARNING **: 14:02:31.320: Failed to load plugin '/home/<username>/amazon-kinesis-video-streams-demos/gst/gst-kvs-plugin/build/_deps/webrtc-src/open-source/lib/libssl.so': /home/<username>/amazon-kinesis-video-streams-demos/gst/gst-kvs-plugin/build/_deps/webrtc-src/open-source/lib/libssl.so: undefined symbol: EVP_idea_cbc, version OPENSSL_1_1_0

(gst-plugin-scanner:447951): GStreamer-WARNING **: 14:02:29.334: Failed to load plugin '/home/<username>/amazon-kinesis-video-streams-demos/gst/gst-kvs-plugin/build/libgstkvsplugin.so': /home/<username>/amazon-kinesis-video-streams-demos/gst/gst-kvs-plugin/build/libgstkvsplugin.so: undefined symbol: signalingClientFetchSync

No such element or plugin 'kvsplugin'
```

**Comment by sauravkumar-volt** [comment]
https://github.com/aws-samples/amazon-kinesis-video-streams-demos/pull/245

**Comment by mariobaldini** [comment]
Maybe this issue is not closed? Since PR #245 was not merged.

I couldn't make `ksvplugin` build and work in Ubuntu 20.04 nor Fedora 36. 
https://github.com/aws-samples/amazon-kinesis-video-streams-demos/pull/245#issuecomment-1478928697

End goal is to have a gst plugin to be used in a `gst-launch-1.0` **for WebRTC**, just like `kvssink` works for video channels.
